src/main/java/org/reactome/addlinks/EnsemblBioMartUtil.java

```diff
@@ -0,0 +1,146 @@
+package org.reactome.addlinks;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class EnsemblBioMartUtil {
```

Comment:
Tip: For classes that _only_ contain static utility methods, some people will create a private no-arg constructor to prevent instantiation of the class. Creating an instances of utility classes isn't "wrong", but it's also somewhat pointless and usually it doesn't even make sense. Adding a private constructor is not a necessary thing to do, but _could_ make it more clear to users of the class that it is a _pure_ utility class that does not need to be instantiated.

https://codepumpkin.com/use-private-constructor-java/#p4

Reply:
Thanks! I'll look into doing that